test/WebSites/ApiExplorerWebSite/Controllers/ApiExplorerResponseTypeWithoutAttributeController.cs

```diff
@@ -10,6 +10,7 @@ namespace ApiExplorerWebSite
     public class ApiExplorerResponseTypeWithoutAttributeController : Controller
     {
         [HttpGet]
+        [ProducesResponseType(typeof(void), 204)]
```

Comment:
Had an in-person discussion. Now you'll get no data (no response types) if you have a void method with no attributes. That seems fine. @kichalla added this here to test the case where you want to specify that it's a 204.